apps/examples/wc-vanilla/main.js

```diff
@@ -2,8 +2,8 @@ import '@justeattakeaway/pie-css';
 import '@justeattakeaway/pie-button';
 import '@justeattakeaway/pie-modal';
 import '@justeattakeaway/pie-icon-button';
```

Comment:
how come this doesn't need a direct path to `/dist/index.js`?

Reply:
For these packages, they each contain only one component so it's okay to import everything from the main entrypoint, which is exactly the file you suggested, but that's read from the package.json and doesn't need to be explicitly stated.